src/interpret.md

```diff
@@ -1,14 +1,13 @@
-# Miri
+# Interpreter
```

Comment:
We could move this under `src/mir`, placing it on the top-level dir seems a bit ambiguous to me.

Reply:
In the TOC it is listed below const-eval so maybe we should make that a folder?